python/ray/serve/tests/test_api.py

```diff
@@ -108,6 +108,46 @@ async def async_d():
     assert ray.get(async_d.get_handle().remote()) == "async!"
 
 
+def test_deploy_sync_function_no_params_call_with_param(serve_instance):
```

Comment:
instead of adding two separate tests, can you just add one test using a class with a sync & and async method?

Reply:
sg